src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function buildLookupTable(ticks, min, max, linear) {
+		var ilen = ticks.length;
```

Comment:
Good catch, will keep it initialized at the top though since I'm using it at line 54.